ext/nnstreamer/tensor_filter/tensor_filter_pytorch.cc

```diff
@@ -378,12 +369,13 @@ TorchCore::getOutputTensorDim (GstTensorsInfo *info)
 /**
  * @brief	process the IValue after forward and extract data from ivalue.
  * @param[in] value IValue containing the output in tensor form
- * @param[out]  output Output tensor memory
+ * @param[out] output Output tensor memory
+ * @param[out] idx index of output
```

Comment:
`idx` seems to be one of input parameters.

Reply:
Thanks! I fixed 'out' with 'in'